Kernel/Term.cpp

```diff
@@ -122,6 +120,9 @@ bool TermList::isSafe() const
   return isVar() || term()->shared();
 }
 
+bool TermList::ground() const 
+{ return !isVar() && term()->ground(); }
```

Comment:
I don't *think* it makes a difference, but why `!isVar()` rather than `isTerm()`?

Reply:
good question :) I was thinking of it as equivalent as well. Can change it.